### PR TITLE
update template version

### PR DESCRIPTION
**Pull request by clarisse-lau**
The previous PR failed with `Error: Version name is a not valid semantic version.` Using `X.Y.Z` version format

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -16,8 +16,8 @@ Metadata:
     ReadmeUrl: "README.md"
     Labels: ["serverless", "synapse", "storage", "S3"]
     HomePageUrl: "https://github.com/Sage-Bionetworks/s3-synapse-sync"
-    SemanticVersion: "1.3"
-    SourceCodeUrl: "https://github.com/Sage-Bionetworks/s3-synapse-sync/tree/1.3"
+    SemanticVersion: "1.2.1"
```
Comment (zaro0508) [inline template.yaml:19]: i suggest going 1.3.1 otherwise we'll all be confused wondering why 1.2.1 is newer than 1.3

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: template.yaml (hunk 1 above)
Comment (clarisse-lau) [reply template.yaml:19]: I haven't actually tagged 1.3 yet since the PR failed (latest is still v1.2), but if it's more clear I can go to 1.3.0

Comment (zaro0508) [reply template.yaml:19]: already merged though so best to go with 1.3.0

Comment (clarisse-lau) [reply template.yaml]: ok, fixed

**Comment by clarisse-lau** [comment]
@zaro0508 Merged this 20 minutes ago, but it looks like the Travis build check did not start? (and I don't see the green check mark that the commit was successful) Should I revert and try again, or is it ok to continue with tagging the current version?

**Pull request merged**